src/rest_api.py

```diff
@@ -132,10 +132,33 @@ def register():
     return flask.jsonify(resp_dict), 200
 
 
-@app.route('/api/v1/report/<repo>')
-def report(repo):
+@app.route('/api/v1/report/<path:repo>/<sha>')
```

Comment:
@abs51295 we have to use query parameters `git-url` and `git-sha` instead of path parameters. The call is going to be like -
```
/report?git-url=<url>&git-sha=<commit>
```